ci/test.lua

```diff
@@ -1,7 +1,7 @@
 -- DFHack developer test harness
 --@ module = true
 
-local expect = require 'test.expect'
+local expect = require 'internal.expect'
```

Comment:
I saw your reasoning in 75da527c9031e3ab202ce3ddaf87057b0b887d29, but I kind of still feel that this filename should have `test` somewhere in its name to reflect the fact that it's only intended to be used for tests. It doesn't have to have `/test/`, though - maybe `test_util/expect.lua`?. It's also not strictly related to the existing `dfhack.internal` module of the same name, so I'd tend away from `internal`.

By the way, where is the luacov configuration that was causing the original issue here? Is it built-in/overridable? Searching for `/test` or `test/` in the DFHack and luacov repos didn't seem to turn up anything related. (Asking to determine if my naming suggestion works and/or is necessary.)

Reply:
The luacov config is totally under our control:
https://github.com/DFHack/scripts/blob/c0e0c307971da9299022cf9c39f3c610c5c0432a/devel/luacov.lua#L95-L100

The intention there is that test files themselves shouldn't contribute to the coverage report.

Reply:
also I moved `expect.lua` from `internal/` to `test_util/`

Reply:
Ah, thanks (I forgot to check there...). Ideally I'd like to have it use the actual path to tests, but communicating that between scripts could get messy since the test path could change at runtime. Checking for `/test/` is totally fine with me.